src/parser.ts

```diff
@@ -279,6 +279,23 @@ export function collapseTerm(tokens: ExprElement[]): ExprElement {
     i -= 1;
   }
 
+  // Detect mixed numbers.
+  for (const [index, currentToken] of tokens.entries()) {
+    if (currentToken instanceof ExprFunction) {
+      const previousToken = tokens[index - 1];
+      if (
+        previousToken instanceof ExprNumber &&
```

Comment:
Should we require here that it is not just a number but an integer in particular?